src/com/facebook/buck/ide/intellij/IjModuleGraphFactory.java

```diff
@@ -315,6 +325,37 @@ public static IjModuleGraph from(
     return new IjModuleGraph(depsBuilder.build());
   }
 
+  private static IjModule createExtraModuleForCompilerOutput(
+      IjModule module, IjProjectConfig projectConfig, ProjectFilesystem projectFilesystem) {
+    Path projectRootPath = Paths.get(projectConfig.getProjectRoot());
+    Path extraModuleBasePath =
+        projectRootPath
+            .resolve("buck-out")
+            .resolve("extra-intellij-output")
```

Comment:
I created it. This is a new folder that we use to store our synthetic library modules so that we don't clutter our project view. The path is arbitrary and can go somewhere else if you prefer